saleor/order/views.py

```diff
@@ -90,6 +90,8 @@ def start_payment(request, order, variant):
         try:
             form = payment.get_form(data=request.POST or None)
         except RedirectNeeded as redirect_to:
+            # NOTE: this is good place to add logic of sending
+            # digital products. Make sure that payment status is confirmed
```

Comment:
Right, it wouldn't work with 3-D Secure - my bad.
Because it depends on users logic we may just delete this note and describe use cases with digital products in documentation